### ISSUE TITLE
update version of mamba on installation

### ISSUE THREAD

**Issue by raybellwaves** [jupyterhub/the-littlest-jupyterhub#797]
There's logic in https://github.com/jupyterhub/the-littlest-jupyterhub/blob/main/tljh/installer.py#L162 for new and old versions of conda and mamba. Can these be replaced with latest conda and mamba?

I understand this could break things with new conda and mamba releases so version here would help

**Comment by ptiwari2407** [comment]
Yes, I am having this issue as well. SO many things are breaking !!

**Comment by minrk** [comment]
tljh is trying to be minimally intrusive, letting admins pick and upgrade versions, rather than forcing downgrades like was done before. If you want to upgrade mamba/conda, you may:

```
mamba update conda mamba
```

at any time. The main change is that tljh no longer automatically upgrades unless things are known to be too old, instead leaving it to admins to control their own environment if they have upgrades or pins they want to enforce.

@ptiwari2407 this isn't a bug report, so I don't think your issue is the same if anything's broken. Feel free to open an issue with logs, errors, etc. if you are having them.
